components/coachmark/index.css

```diff
@@ -10,127 +10,157 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+	--spectrum-coachmark-title-font-style: var(--spectrum-heading-serif-font-style);
+	--spectrum-coachmark-title-text-font-weight: var(--spectrum-heading-sans-serif-font-weight);
+	--spectrum-coachmark-title-font-size: var(--spectrum-coach-mark-title-size);
+	--spectrum-coachmark-title-text-line-height: var(--spectrum-heading-line-height);
+
+	--spectrum-coachmark-content-font-color: var(--spectrum-body-color);
+	--spectrum-coachmark-content-font-weight: var(--spectrum-body-sans-serif-font-weight);
+	--spectrum-coachmark-content-font-family: var(--spectrum-body-sans-serif-font-family);
+	--spectrum-coachmark-content-font-style: var(--spectrum-body-sans-serif-font-style);
+	--spectrum-coachmark-content-line-height: var(--spectrum-body-line-height);
+	--spectrum-coachmark-content-font-size: var(--spectrum-coach-mark-body-size);
+
+	--spectrum-coachmark-step-color: var(--spectrum-coach-mark-pagination-color);
+	--spectrum-coachmark-step-font-weight: var(--spectrum-body-medium-font-weight);
+	--spectrum-coachmark-step-font-family: var(--spectrum-body-sans-serif-font-family);
+	--spectrum-coachmark-step-font-style: var(--spectrum-body-sans-serif-font-style);
+	--spectrum-coachmark-step-line-height: var(--spectrum-body-line-height);
+	--spectrum-coachmark-step-font-size: var(--spectrum-coach-mark-pagination-body-size);
+	--spectrum-coachmark-step-to-bottom: var(--spectrum-coach-mark-pagination-text-to-bottom-edge);
+
+	/* mods */
+	--mod-buttongroup-justify-content: flex-end;
+	--mod-popover-border-width: var(--spectrum-border-width-100);
+	--mod-popover-corner-radius: var(--spectrum-corner-radius-100);
+	--mod-popover-content-area-spacing-vertical: 0;
+	--mod-button-edge-to-visual-only: 9px;
+}
 
-.spectrum-CoachMarkPopover {
+.spectrum-CoachMark {
   position: relative;
-
-  min-inline-size: var(--spectrum-coachmark-min-width);
-  max-inline-size: var(--spectrum-coachmark-max-width);
-
-  border-radius: var(--spectrum-coachmark-border-radius);
-  border-width: var(--spectrum-coachmark-border-size);
-  border-style: solid;
-
+	min-inline-size: var(--mod-coachmark-min-width, var(--spectrum-coachmark-min-width));
+	max-inline-size: var(--mod-coachmark-max-width, var(--spectrum-coachmark-max-width));
+	inline-size: var(--mod-coachmark-width, var(--spectrum-coachmark-width));
 }
-.spectrum-CoachMarkPopover-image {
-  border-radius: var(--spectrum-coachmark-border-radius) var(--spectrum-coachmark-border-radius) 0 0;
 
-  inline-size: 100%;
-}
+.spectrum-CoachMark-buttongroup {
+	display: var(--spectrum-coachmark-buttongroup-display);
 
-.spectrum-CoachMarkPopover-header,
-.spectrum-CoachMarkPopover-content,
-.spectrum-CoachMarkPopover-footer {
-  padding-block: 0;
-  padding-inline: var(--spectrum-coachmark-padding-x);
+	&--mobile {
+ 		--mod-buttongroup-spacing-horizontal: var(--spectrum-spacing-100);
+		display: var(--spectrum-coachmark-buttongroup-mobile-display);
```

Comment:
Is there any reason that `--spectrum-coachmark-buttongroup-mobile-display` is `display:none` ?

Reply:
In mobile we also need `display:flex`

Reply:
These values [from custom tokens]( https://github.com/adobe/spectrum-css/blob/6e95f44b997b838f5b5fdaed5d71cf10e2ce124b/components/tokens/custom-spectrum/custom-medium-vars.css#L73C3-L73C3) and have different values for Large Scale (Mobile) and Medium Scale (Desktop). 

Both the mobile and desktop button-groups are in the html and we are toggling the display with custom tokens. So `--spectrum-coachmark-buttongroup-mobile-display` is `display:none` as default or desktop and will change to flex when displayed on Scale Large (aka Mobile).

Reply:
I think maybe the term mobile is confusing things, as far as the CSS is concerned it should be just scale differences with `--mobile` being what is displayed when the scale is Large. Are we able to rely on the custom tokens toggling display taking that into consideration?